pkg/backup/abs/absconfig/abs_ctx.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The etcd-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package absconfig
+
+// ABSContext represents the controller context around a ABS backend configuration
+type ABSContext struct {
```

Comment:
I don't think `ABSContext` is needed since it is never populated from anywhere; We are not parsing any `ABS` flags from etcd-operator binary.

All the ABS information should be passed in by cluster spec via `spec.BackupPolicy`.

 cc/ @hongchaodeng for confirmation.

Reply:
Okay, will wait for confirmation before removing...